tools/check-bash-scripts

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
+"${1:-shfmt}" -d .
```

Comment:
I don't understand why you create this wrapper and your git commit message in https://github.com/os-autoinst/os-autoinst/pull/2450/commits/d20aa3a8e931bb2ca65030b7043e2b00a0df7b46 does not tell more about that context. Please put more details in the git commit message and remove that "Signed-off-by" line, that's redundant.

For this specific change here I assume you copied "test-local-perl-style" but if you take a look further like in "find_program(SPELL_PATH spell)" how an external command is used

Reply:
To me it looks like a copy from the shellcheck check, and every check seems to have a wrapper, so I guess I would have created one myself.

Reply:
@Martchus you might be able to give some insight on this design with lots of wrappers here

Reply:
I guess the wrappers were because someone wanted a simple shell script that can be invoked separately. I personally don't think it is that useful and probably would invoke shfmt just from CMake (so `ctest` can be used to invoke the test). However, considering we already have one wrapper and we have people who might not be that experienced with CMake I guess creating one more wrapper is not a big deal.

Note that in CMake-based build scripts it is a best practice to make paths of executables overridable; that's why `find_program(SH_PATH shfmt)` is used here and that's why we pass that path to the wrapper script. I guess we shouldn't needlessly deviate from that practice.

By the way, the corresponding CMake code (which is actually the lengthy part) could likely be reduced by using a loop but it would be at the cost of readability.